### PR TITLE
PP-717: Change PBS package RPM spec file hwloc version dependency to 1.11 or greater

### PR DESCRIPTION
**Pull request by visheshh**
PP-717: Change PBS package RPM spec file hwloc version dependency to 1.11 or greater

Using hwloc version greater than 1.11 to help us count Packages , MIC's , Change Socket name to Package as per the new hwloc library in server and pbs_topologyinfo .

#### Issue ID

* **[PP-717](https://pbspro.atlassian.net/browse/PP-717)**

#### Problem description
* New hwloc library has renamed Socket to Package 

#### Cause / Analysis
* Need to count Packages instead of Socket

#### Solution description
* Changed Socket to Package in node_func.c and pbs_topologyinfo.py , Updated hwloc build requirements to use hwloc version greater than 1.11 .

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [x] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [ ] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [x] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to the Jira ticket** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: .travis/opensuse.sh [hunk 1]
```diff
@@ -7,6 +7,8 @@ ${DOCKER_EXEC} zypper -n install time.x86_64
 ${DOCKER_EXEC} zypper -n ref
 ${DOCKER_EXEC} zypper -n install rpmdevtools
 ${DOCKER_EXEC} rpmdev-setuptree
+${DOCKER_EXEC} zypper -n ar -f -G http://ftp.pbone.net/mirror/ftp5.gwdg.de/pub/opensuse/repositories/Virtualization/openSUSE_Leap_42.1/Virtualization.repo
+${DOCKER_EXEC} zypper -n ref
```
Comment (mike0042) [inline .travis/opensuse.sh]: Are we expecting open source consumers to add this repo to their zypper configuration? That could cause a great deal of confusion, especially for less experienced users. I don't think we should be adding any requirements that don't come from the official OpenSUSE repos.

**Comment by visheshh** [comment]
Hi Mike ,
I apologize i did a force update by mistake , I have updated repo to a Official open suse repo which has hwloc 1.11 . Can we use this and document for OSS users to use this repo.
Thanks,
Vishesh

File: .travis/opensuse.sh [hunk 2]
```diff
@@ -7,6 +7,8 @@ ${DOCKER_EXEC} zypper -n install time.x86_64
 ${DOCKER_EXEC} zypper -n ref
 ${DOCKER_EXEC} zypper -n install rpmdevtools
 ${DOCKER_EXEC} rpmdev-setuptree
+${DOCKER_EXEC} zypper -n ar -f -G http://download.opensuse.org/repositories/Virtualization/${OS_NAME_VERSION}/Virtualization.repo
```
Comment (mike0042) [inline .travis/opensuse.sh]: I'm a bit confused here. For OpenSUSE it looks like Tumbleweed is the first release to support hwloc-1.11. Older releases provide older versions. For SLE12 it looks like they are already up to hwloc-2.0. Don't we have to support versions of hwloc both before and after 1.11 in order to support a variety of SUSE platforms? That means the PBS Pro code that parses the output needs to be tolerant of both "Package" and "Socket".

**Comment by visheshh** [comment]
Ticket is closed as Wont do.

**Pull request closed without merging**